python/tests/core/constraints/factories/test_count_metrics.py

```diff
@@ -1,10 +1,21 @@
 from whylogs.core.constraints.factories import (
     count_below_number,
+    is_complete,
```

Comment:
Consider renaming this something more explicit like "no_nulls" or "no_missing_values"

"is_complete" works but its broad enough one might expect that such a check is for completeness in terms of an expected row count or values in a range, or that there isn't any value missing in a set (e.g. the data contains values for all entries in a set).

Reply:
done!